GNUmakefile

```diff
@@ -150,6 +150,7 @@ CFLAGS += -Wall -Werror -Wshadow -Wextra -Iinc \
           -D__STRICT_ANSI__ \
           -D_ISOC99_SOURCE \
           -D_POSIX_C_SOURCE=200112L \
+		  -DOPENSSL_API_COMPAT=0x10100000L \
```

Comment:
This worked fine for all the platforms that I built.

However, iRODS itself is not accepting this syntax for some platforms (specifically, almalinux:8, ubuntu:18.04, and debian:11). I had to use a different syntax to build iRODS packages for all platforms:
```make
-DOPENSSL_API_COMPAT=0x101000 \
```
This is acceptable according to the OpenSSL documentation. Should we use a similar syntax here, or just go with the current one which I have confirmed (i.e. the `0x10100000L` syntax)?

Reply:
Actually, this "second form" is recommended in the docs. I'll change it here too.

Reply:
Done